embedded/app/src/states.c

```diff
@@ -37,7 +37,7 @@ extern stateTransition_t *findTransition(state_t *currState, char *name);
  */
 
 static bool checkStopped(void) {
-	return fabs(data->motion->vel) < MAX_STOPPED_VEL &&  (time(NULL) - data->timers->lastRetro) > 15;
+	return fabs(data->motion->vel) < MAX_STOPPED_VEL &&  (getuSTimestamp() - data->timers->lastRetro) > 15000000;
```

Comment:
Can we make the 15/15000000 a const and not a floating number? (This is my err from earlier but we can get it fixed now)